s32193-SlawomirRudnicki/zadaniaTydzien3.txt

```diff
@@ -0,0 +1,86 @@
+zadanie 0
+
+1. uzylem komendy cd .. aby cofnac sie do katalogu home 
+	s32193@szuflandia22:/home$ pwd
+	/home
+2. komenda ls -l wypisala 6 katalogow
+3. wrocilem  do swojego katalogu z katalogu home
+4. ls -l po raz drugi: 6 folderow 10 plikow
+5. wyszukalem wszystkie pliki oraz foldery zawierajace litere "d" 
+	komenda znalazla 6 nazw zawierajacych ta litere
+6. uzylem komendy ls -a | grep d > dlista.txt zeby utworzyc plik tekstowy z
+wynikiem polecania grep d
+7. uzylem komendy cat dlista.txt aby odczytac co jest w tym pliku zapisane 
+	| przekazuje output jednej operacji do drugiej
+	> przekazuje output operacji do pliku
+8. uzylem komendy ls -a | grep d | tee dlista2.txt
+	otrzymalem plik tekstowy jak w przypadku poprzednim, z ta roznica ze output zostal rowniez
+wyswietlony
+9. komenda ls -a | grep d | dlista3.txt zglasza blad "nie znaleziono polecania"
+program tee przekazuje output ktory otrzymala do wyjscia standardowego (na ekran) oraz do pliku
+
+zadanie 1
+
+1. komenda ps wyswietlila 2 procesy, bash i sama siebie
+ s32193@szuflandia22:~$ ps
+    PID TTY          TIME CMD
+3106562 pts/14   00:00:00 bash
+3119937 pts/14   00:00:00 ps
+
+2. komenda ps -a wyswietlila 1 proces, ps
+komenda ps x wyswietlila cala grupe procesow
+komenda ps -axu wyswietlila duza liste procesow
+znak zapytania oznacza proces nie nalezacy do grupy posiadajacej TTY, 
+"x" sciaga restrykcje "must have a tty" -> man ps
+grupa procesow posiadajacych tty to procesy ktore sa powiazane z terminalem na ktorym jest uzyta komenda ps
+wiec procesy z ? sa procesami ktore nie sa powiazane z terminalem na ktorym komenda ps zostala wywolana
+zrodlo -> manual
+
+3. wyswietlilem procesy bash:
+	ps -axu | grep bash
```

Comment:
Jeśli będzie user o loginie `barabash` i on będzie miał uruchomiony jakiś proces nie będący bashem to i tak zostaną jego wszystkie procesy wypisane z tytułu słowa bash w loginie. To rozwiązanie jest do poprawy. Jakiego przełacznika trzeba użyć by zadanie było rozwiązanie poprawnie?